src/lib/components/questions/exam-core.svelte.ts

```
@@ -0,0 +1,597 @@
+		draftSelections = Array.from({ length: count }, () => null);
+		answers = Array.from({ length: count }, () => null);
+		flaggedIndexes = [];
+		failedIndexes = [];
+		currentIndex = 0;
+		requestedCount = count;
+	}
+
+	function beginTiming(limitMs: number | null | undefined): void {
+		const now = Date.now();
+		startedAtMs = now;
+		startedAtIso = new Date(now).toISOString();
```

ESLint

[error] Found a mutable instance of the built-in Date class. Use SvelteDate instead.

(svelte/prefer-svelte-reactivity)